proposals/dao/fip_60.ts

```diff
@@ -0,0 +1,255 @@
+import { ethers } from 'hardhat';
+import chai, { expect } from 'chai';
+import CBN from 'chai-bn';
+import {
+  DeployUpgradeFunc,
+  NamedContracts,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import { deploy as deploySTW } from '@scripts/deploy/deployStakingTokenWrapper';
+import { getImpersonatedSigner } from '@test/helpers';
+import { forceEth } from '@test/integration/setup/utils';
+
+chai.use(CBN(ethers.BigNumber));
+
+const UNDERLYINGS = [
+  '0xBaaa1F5DbA42C3389bDbc2c9D2dE134F5cD0Dc89',
+  '0x06cb22615BA53E60D67Bf6C341a0fD5E718E1655',
+  '0x3D1556e84783672f2a3bd187a592520291442539'
+];
+const NAMES = ['FeiRari d3pool Fuse', 'FeiRari FEI-3Crv Metampool Fuse', 'FeiRari Gelato FEI-DAI LP Fuse'];
+const SYMBOLS = ['fD3-8', 'fFEI-3Crv-8', 'fG-UNI-FEI-DAI-8'];
+
+const COLLATERAL_FACTOR = ethers.constants.WeiPerEther.mul(60).div(100); // 60% CF
+
+const IMPL = '0x67Db14E73C2Dce786B5bbBfa4D010dEab4BBFCF9';
```

Comment:
What is IMPL? Need more context here.

Reply:
the impl of the CErc20. Its a relic I'll clean it up